ParseLiveQuery/src/main/java/com/parse/ParseLiveQueryClient.java

```diff
@@ -21,10 +20,18 @@ public static ParseLiveQueryClient getClient() {
             return new ParseLiveQueryClientImpl();
         }
 
+        public static ParseLiveQueryClient getClient(WebSocketClientFactory webSocketClientFactory) {
```

Comment:
added a few more interface methods (so you don't need to specify the Uri thanks to changes to @hermanliang